ephios/event_management/templates/event_management/fragments/shift_box_big.html

```diff
@@ -41,28 +41,22 @@ <h5 class="card-title">
                                                      <p>{{ error.message }}</p>
                                                  {% endfor %}"
                                                 {% endif %}>
-                                            <button type="submit" class="btn btn-outline-success"
-                                                    name="signup_choice" value="sign_up"
-                                                    {% if not can_sign_up %}
-                                                    style="pointer-events: none;" disabled
-                                                    {% endif %}>
+                                            <button type="submit" name="signup_choice" value="sign_up"
+                                                    {% if not can_sign_up %}class="btn btn-outline-success btn-disabled" disabled>{% else %}class="btn btn-outline-success btn-disabled">{% endif %}
```

Comment:
Please review this line. You have `btn-disabled` in both cases. Also, lets remove the `not` to avoid confusion. (same for decline)